parse.cpp

```diff
@@ -0,0 +1,189 @@
+#include<iostream>
+#include<cstring>
+#include<fstream>
+
+using namespace std;
+
+class ParsePAT {
+    public:  	
```

Comment:
If every member is public then why class is needed. Can you please go through and understand encapsulation and abstraction?

Reply:
Ok, I will go through ,I will do the necessary changes.